source/postprocess/topography.cc

```diff
@@ -187,6 +223,13 @@ namespace aspect
                              "Units: years if the "
                              "'Use years in output instead of seconds' parameter is set; "
                              "seconds otherwise.");
+          prm.declare_entry ("Analytical solution of example", "1",
```

Comment:
This is not a clean solution. Can you make a new plugin that sits on the benchmark folder, instead?